components/project-loader/project-loader.jsx

```diff
@@ -18,7 +19,12 @@ export default React.createClass({
     };
   },
   componentDidMount() {
-    this.fetchData();
+    if (pageSettings.restore) {
```

Comment:
we should probably call this `shouldRestore` rather than `restore`, as the latter is a verb and so suggests this is a function that should be called.